infra/examples-dev/gcp/README.md

```diff
@@ -0,0 +1,73 @@
+# example-google-workspace
+** alpha quality example atm **
+
+A Terraform root module to provision GCP project for Psoxy, configure it, and create necessary infra
+to for connections to all supported Google Workspace sources, with state stored to local filesystem.
+As such, it is not appropriate for scenario with multiple developers. As state will contain
+sensitive information (eg, service account keys), care should be taken in production to ensure that
+the filesystem in question is secure or another Terraform backend should be used (eg, GCS bucket
+encrypted with a CMEK).
+
+## Usage
+
+### Prereqs
+
+You'll need:
+- a Bash-like shell environment on Linux, MacOS, or [WSL on Windows](https://learn.microsoft.com/en-us/windows/wsl/install).
+- [`git` installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), although it
+  is usually included on those platforms (check with `git --version`).
+- an AWS account and credentials, as described in [Psoxy's AWS - Getting Started docs](https://github.com/Worklytics/psoxy/blob/v0.4.17/docs/aws/getting-started.md)
```

Comment:
point links to `main` so they go to latest version always?
```suggestion
- an AWS account and credentials, as described in [Psoxy's AWS - Getting Started docs](https://github.com/Worklytics/psoxy/blob/main/docs/aws/getting-started.md)
```

Reply:
why AWS is needed at all? Isn't this the GCP example?

Reply:
Also, the GCP project is not created, it should exist, right? Shouldn't it be listed as a pre-req too?

Reply:
well, imho it's like an interface - if we move stuff around the link will break

re: AWS, yes should be changed to gcp